fixtures/pytest_store.py

```diff
@@ -89,51 +89,51 @@ def base_url(self):
             else, the base_url from the config is returned."""
         return self.current_appliance.url
 
+    def _maybe_get_plugin(self, name):
+        """ returns the plugin if the pluginmanager is availiable and the plugin exists"""
+        return self.pluginmanager and self.pluginmanager.getplugin(name)
```

Comment:
```property_or_none``` used to catch the ```AttributeError``` but I don't see it being caught here. It also used to return ```None```, but won't this return ```False```?